### PR TITLE
[someone please finish this for me I'm literally out of git directories] Gives security lockers non-lethal riot control baseball bats

### PR DESCRIPTION
**Pull request by silicons**
Bwahhahaha they thought I was joking.

20 stamina/hit
6.5 stamina to swing.
fits in backpack.

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -500,13 +500,27 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
 
+/obj/item/melee/baseball_bat/riot
+	name = "riot control bat"
+	desc = "A new addition to DonkCo's classic \"riot control\" line of non-lethal deterrants. Completely nonlethal, but hurts like a bitch. <span class='bold'>Quite hefty, though. Manage your stamina.</span>"
+	damtype = STAMINA
+	icon_state = "riot_bat"
+	item_state = "riot_bat"
+	force = 20
+	mass = 6.5		//no infinite stunlock/batonning please!
+	w_class = WEIGHT_CLASS_MEDIUM
```
Comment (Ghommie) [inline code/game/objects/items/weaponry.dm]:
```suggestion
	w_class = WEIGHT_CLASS_NORMAL
```

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -500,13 +500,27 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
 	homerun_able = 1
 
+/obj/item/melee/baseball_bat/riot
+	name = "riot control bat"
+	desc = "A new addition to DonkCo's classic \"riot control\" line of non-lethal deterrants. Completely nonlethal, but hurts like a bitch. <span class='bold'>Quite hefty, though. Manage your stamina.</span>"
+	damtype = STAMINA
+	icon_state = "riot_bat"
+	item_state = "riot_bat"
+	force = 20
+	mass = 6.5		//no infinite stunlock/batonning please!
```
Comment (Ghommie) [inline code/game/objects/items/weaponry.dm]:
```suggestion
	total_mass = 6.5		//no infinite stunlock/batonning please!
```

File: code/game/objects/items/weaponry.dm [hunk 3]
```diff
@@ -500,13 +500,27 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
 	total_mass = 2.7 //a regular wooden major league baseball bat weighs somewhere between 2 to 3.4 pounds, according to google
 
 /obj/item/melee/baseball_bat/homerun
 	name = "home run bat"
 	desc = "This thing looks dangerous... Dangerously good at baseball, that is."
 	homerun_able = 1
 
+/obj/item/melee/baseball_bat/riot
+	name = "riot control bat"
+	desc = "A new addition to DonkCo's classic \"riot control\" line of non-lethal deterrants. Completely nonlethal, but hurts like a bitch. <span class='bold'>Quite hefty, though. Manage your stamina.</span>"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weaponry.dm:514]: DonkCo tech is illegal. Syndacat tech

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Lore brakes

**Pull request closed without merging**